components/core/src/package/archive.rs

```diff
@@ -204,9 +214,10 @@ impl PackageArchive {
     // See https://rust-lang.github.io/rust-clippy/master/index.html#wrong_self_convention
     #[allow(clippy::wrong_self_convention)]
     pub fn is_a_service(&mut self) -> bool {
-        match self.svc_user() {
-            Ok(_) => true,
-            _ => false,
+        if let Some(_user) = self.svc_user() {
+            true
+        } else {
+            false
```

Comment:
This `if/else` could now be simplified as `self.svc_user().is_some()`.

Reply:
`ERR_SOTRUE` I've updated and pushed in https://github.com/habitat-sh/habitat/pull/7814/commits/a4d2b8a23a096374c3b8128eb816f71bd68b8d5a